dsc_lib/src/extensions/dscextension.rs

```diff
@@ -22,6 +23,8 @@ pub struct DscExtension {
     pub version: String,
     /// The capabilities of the resource.
     pub capabilities: Vec<Capability>,
+    /// The extensions supported for importing.
+    pub import_extensions: Option<Vec<String>>,
```

Comment:
I think extension is getting overloaded - maybe `import_file_extensions` or something like that would be more clear?